tests/test_uint_muldiv.nim

```diff
@@ -30,8 +30,16 @@ suite "Testing unsigned int multiplication implementation":
     let b = 1_000_000_000.stuint(64)
     let c = 1_000.stuint(64)
 
-    check: cast[uint64](a*b*c) == 1_000_000_000_000_000_000_000'u64 # need 70-bits
+    let x = 1_000_000_000'u64
+    let y = 1_000_000_000'u64
+    let z = 1_000'u64
 
+    {.push overflowChecks: off.}
```

Comment:
this is a bit strange - uint64 shouldn't have overflow checks (it should wrap)

Reply:
it doesn't, only a bit paranoid